tests/shared/units.spec.js

```diff
@@ -58,4 +58,33 @@ describe('Shared: Units', function () {
       m.chai.expect(units.bytesToMegabytes(332000)).to.equal(0.332)
     })
   })
+
+  describe('.millisecondsToSeconds()', function () {
+    it('should convert milliseconds to seconds', function () {
+      const seconds = units.millisecondsToSeconds(10000)
+      m.chai.expect(seconds).to.equal(10)
+    })
```

Comment:
Would it be worth also adding a test for a not-exact number of seconds, e.g. 5500 milliseconds?

Reply:
Nah, timers run in ms intervals anyways, 500 more or less shouldn't matter.

Reply:
Nah, I meant specifically for this `millisecondsToSeconds` unit-test, i.e. add a check that `units.millisecondsToSeconds(5500) == 5.5`